docs/code-reviews.md

```diff
@@ -1,16 +1,16 @@
 ## What is a code review?
 
 A code review is a critical quality assurance practice. It involves systematically
-checking a fellow programmer's (or researcher's) code for errors or possible 
+checking a fellow programmer's (or researcher's) code for potential errors or possible 
 improvements. 
 
 It is an everyday activity in the software development world and improves
-both quality and speed in programming. 
+both quality and speed in programming. Code reviews are standard practice within OpenSAFELY studies and the default should be to always get your code reviewed. 
 
 ## How to ask for a review
 
 Github has a feature built into the workflow that allows easy code review with collaborators, and we would recommend 
-that you use this tool. 
+that you use this tool. [Github](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews) has a some documentation on Pull Requests and Code Reviews which we recommend for background reading. 
```

Comment:
wdyt about moving the link from the `github` to `docs on prs`, like this:
```
Github has some [documentation on Pull Requests and Code Reviews]((https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews) which we recommend for background reading.
```

Reply:
good idea - changed